.github/workflows/linkcheck.yml

```
@@ -0,0 +1,39 @@
+name: Link Check
+
+on:
+  schedule:
+    # Run every Monday at 8am to check URL links still resolve
+    - cron: "0 8 * * MON"
+
+jobs:
+  docs:
+    strategy:
+      fail-fast: false
+      matrix:
+        os: ["ubuntu-latest"]
+        python: ["3.10"]
+
+    runs-on: ubuntu-latest
```

Either 
```suggestion
    runs-on: ${{ matrix.os }}
```
or delete matrix.os

---

deleted os